### PR TITLE
Validate PHNX

### PR DESCRIPTION
**Pull request by julesdamasta**
Validation for Phoenix

# Validate [{PHNX}](https://solscan.io/token/JAzesW3tU2VLDx99pwgkDMca2DTNsEhSWG8sj21B1niz#holders)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/Phnxsol/status/1699083433274720473
- Coingecko/ CMC URL (If available): n/a

## Validation (Please check off boxes):
- [y] The metadata provided in the PR matches what is on-chain (Mandatory)
- [y] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [n] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [y] My change matches the format in the file (no spaces between fields).
- [y] My token is already live and trading on Jupiter.
- [y] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by wrstnghtmre** [comment]
> Validation for Phoenix
> 
> # Validate [{PHNX}](https://solscan.io/token/JAzesW3tU2VLDx99pwgkDMca2DTNsEhSWG8sj21B1niz#holders)
> ## Attestations (Please provide links):
> * Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/Phnxsol/status/1699083433274720473
> * Coingecko/ CMC URL (If available): n/a
> 
> ## Validation (Please check off boxes):
> * [y] The metadata provided in the PR matches what is on-chain (Mandatory)
> * [y] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
> * [n] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
> 
> ## Acknowledgement (Please check off boxes)
> * [y] My change matches the format in the file (no spaces between fields).
> * [y] My token is already live and trading on Jupiter.
> * [y] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

Please validate $phnx.  Wen?  Wen validate $phnx

**Comment by wrstnghtmre** [comment]
Wen? Wen validate $phnx. Please validate $phnx

File: validated-tokens.csv [hunk 1]
```diff
@@ -766,3 +766,4 @@ Blaze,BLZE,BLZEEuZUBVqFhj8adcCFPJvPVCiCyVmh3hkJMrU8KuJA,9,https://solblaze.org/a
 COCO Token,COCO,74DSHnK1qqr4z1pXjLjPAVi8XFngZ635jEVpdkJtnizQ,5,https://shdw-drive.genesysgo.net/EV1ARo89dwRzR1kv7JMr7V97qrcXjffkcwEuNHMJfJmz/COCO_icon.png,true
 Quantity build,QB,4iGtA8VSTFNFjXdQZiSkMkM2sS4x21tC9y5DAU9WxoQd,5,https://arweave.net/Pg5DdUVvKwY6iaqFMsXwomg7H40CFzHyG14E7PJyn9s,true
 EUROe Stablecoin,EUROe,2VhjJ9WxaGC3EZFwJG9BDUs9KxKCAjQY4vgd1qxgYWVg,6,https://dev.euroe.com/persistent/token-icon/png/256x256.png,true
+Phoenix,PHNX,JAzesW3tU2VLDx99pwgkDMca2DTNsEhSWG8sj21B1niz,6,https://i.imgur.com/R6E8S1h.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:769]:
```suggestion
Phoenix,PHNX,JAzesW3tU2VLDx99pwgkDMca2DTNsEhSWG8sj21B1niz,6,https://bafybeicmwtzskjhcsnoymhsv5fmlruwoxgounmcweiei2qnxh6ayesupiq.ipfs.nftstorage.link,true
```

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

**Comment by wrstnghtmre** [comment]
Validate $phnx

**Pull request merged**